change/@fluentui-react-charting-acfa7b51-4c62-47b3-b7fd-09a97c5e9d67.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29395&commitId=84da4ec571882ee8969c3e92e0298c482d51af2a&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|3714| Changed|
|react-charting-AreaChart.Multiple.hover.chromium.png|4282| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>